src/data/set/intervals/basic.lean

```diff
@@ -155,21 +155,21 @@ nonempty.to_subtype nonempty_Ioi
 instance nonempty_Iio_subtype [no_bot_order α] : nonempty (Iio a) :=
 nonempty.to_subtype nonempty_Iio
 
-@[simp] lemma Ioo_eq_empty (h : b ≤ a) : Ioo a b = ∅ :=
-eq_empty_iff_forall_not_mem.2 $ λ x ⟨h₁, h₂⟩, (h₁.trans h₂).not_le h
+@[simp] lemma Icc_eq_empty (h : ¬a ≤ b) : Icc a b = ∅ :=
+eq_empty_iff_forall_not_mem.2 $ λ x ⟨ha, hb⟩, h (ha.trans hb)
 
-@[simp] lemma Ico_eq_empty (h : b ≤ a) : Ico a b = ∅ :=
-eq_empty_iff_forall_not_mem.2 $ λ x ⟨h₁, h₂⟩, (h₁.trans_lt h₂).not_le h
+@[simp] lemma Ico_eq_empty (h : ¬a < b) : Ico a b = ∅ :=
+eq_empty_iff_forall_not_mem.2 $ λ x ⟨ha, hb⟩, h (ha.trans_lt hb)
 
-@[simp] lemma Icc_eq_empty (h : b < a) : Icc a b = ∅ :=
-eq_empty_iff_forall_not_mem.2 $ λ x ⟨h₁, h₂⟩, (h₁.trans h₂).not_lt h
+@[simp] lemma Ioc_eq_empty (h : ¬a < b) : Ioc a b = ∅ :=
+eq_empty_iff_forall_not_mem.2 $ λ x ⟨ha, hb⟩, h (ha.trans_le hb)
 
-@[simp] lemma Ioc_eq_empty (h : b ≤ a) : Ioc a b = ∅ :=
-eq_empty_iff_forall_not_mem.2 $ λ x ⟨h₁, h₂⟩, (h₂.trans h).not_lt h₁
+@[simp] lemma Ioo_eq_empty (h : ¬a < b) : Ioo a b = ∅ :=
+eq_empty_iff_forall_not_mem.2 $ λ x ⟨ha, hb⟩,  h (ha.trans hb)
```

Comment:
I think it might be worth keeping the old lemmas too, as they look like they're better `simp` lemmas.